pmd-java/src/main/java/net/sourceforge/pmd/lang/java/multifile/ClassStats.java

```diff
@@ -98,9 +98,9 @@ void addField(String name, JavaFieldSignature sig) {
     @Override
     public boolean hasMatchingOpSig(String name, JavaOperationSigMask mask) {
         // Indexing on signatures optimises this type of request
-        for (JavaOperationSignature sig : operations.keySet()) {
-            if (mask.covers(sig)) {
-                if (operations.get(sig).contains(name)) {
+        for (Entry<JavaOperationSignature, Set<String>> entry : operations.entrySet()) {
+            if (mask.covers(entry.getKey())) {
+                if (operations.get(entry.getKey()).contains(name)) {
```

Comment:
This is wrong, should be `entry.getValue()`

Reply:
Fixed